api.json

```diff
@@ -0,0 +1,761 @@
+              "modifiedAt": "String",
+              "meetings": [{
+                "meetingId": "Number",
+                "meetingType": "String",
+                "outreachPartner": "String",
+                "createdAt": "String",
+                "modifiedAt": "String"
+              }]
+            }]
+          }
+        }
+      }
+    },
+    "/users": {
+      "description": "",
+      "/": {
+        "GET": {
+          "description": "Retrieve all users accessible with the given scopes",
+          "auth": true,
+          "scope": [],
+          "query": {
+            "fields": ["String"]
+          },
+          "response": {
+            "id": "Number",
+            "name": "String",
+            "gender": "String",
+            "birthYear": "String",
+            "email": "String",
+            "phoneNumber": "String",
+            "postCode": "String",
+            "emailConsent": "Boolean",
+            "smsConsent": "Boolean",
+            "qrCodeUrl": "String",
+            "registeredAt": "String",
+            "lastActiveAt": "String",
+            "modifiedAt": "String"
+          }
+        },
+        "POST": {}
+      },
+      "/register": {
+        "POST": {
+          "auth": "Maybe - orgs/volunteers don't need auth but visitors do",
+          "scope": [],
+          "body": {
+            "organisationId": "Number",
+            "name": "String",
+            "gender": "String",
+            "birthYear": "String",
+            "email": "String",
+            "phoneNumber": "String",
+            "postCode": "String",
+            "emailConsent": "Boolean",
+            "smsConsent": "Boolean"
+          },
+          "response": {
+            "id": "Number",
+            "name": "String",
+            "gender": "String",
+            "birthYear": "String",
+            "email": "String",
+            "phoneNumber": "String",
+            "postCode": "String",
+            "emailConsent": "Boolean",
+            "smsConsent": "Boolean",
+            "qrCodeUrl": "String",
+            "registeredAt": "String"
+          }
+        }
+      },
+      "/login": {
+        "POST": {
+          "auth": null,
+          "scope": null,
+          "body": {},
+          "response": {}
+        }
+      },
+      "/:id||me": {
+        "GET": {
+          "description": "Retrieve details of a specific user",
+          "auth": true,
+          "scope": {},
+          "query": {
+            "fields": [],
+          },
+          "response": {
+            "id": "Number",
+            "name": "String",
+            "qrSecret": "String",
+            "regionId": "Number",
+            "sectorId": "Number",
+            "organisationId": "Number",
+            "gender": "String",
+            "birthYear": "String",
+            "email": "String",
+            "phoneNumber": "String",
+            "emailConsent": "Boolean",
+            "smsConsent": "Boolean",
+            "qrCode": "String",
```

Comment:
I think either this should be a conditional field based on whether the user is a visitor or not, or it should be a field that's only returned when we query explicitly for visitors, either through a "user type" filter or an alias like `/visitors/...`